crates/uniffi/uniffi_SPLIT_ME/src/profile/v100/entity/persona/persona.rs

```diff
@@ -22,7 +22,7 @@ decl_vec_samples_for!(Personas, Persona);
 /// technically, nothing stops a user from building their own wallet and uploading
 /// personal information to the metadata of the Identity component... but `Sargon`
 /// never will, nor will the Radix Wallet.).
-#[derive(Clone, PartialEq, Hash, Eq, InternalConversion, uniffi::Record)]
```

Comment:
`Persona`/`Account` now contains `base: BaseEntity` field which shares all common fields between those entity types, and in rust they impl Deref<Target = BaseEntity> for Account/Persona` making it as if the fields of `BaseEntity` were fields "flat" into each entity type. this is ofc not possible with Uniffi... so we remain the old structure and have to manually impl `From`.